GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -25,5 +25,8 @@ public class LoggingBroker : ILoggingBroker
 
         public void LogWarning(string message) =>
             this.logger.LogWarning(message);
+            
+        public void LogError(Exception exception) =>
+           this.logger.LogError(exception, exception.Message);
```

Comment:
Add a parameter alias

```suggestion
        public void LogError(Exception exception) =>
           this.logger.LogError(exception, message: exception.Message);
```

Reply:
@cjdutoit  ok